src/libraries/System.Text.Json/tests/Serialization/CustomConverterTests/CustomConverterTests.Object.cs

```diff
@@ -311,6 +311,72 @@ public override void Write(Utf8JsonWriter writer, object value, JsonSerializerOp
             }
         }
 
+        private class PrimitiveConverter : JsonConverter<object>
+        {
+            public override bool CanConvert(Type typeToConvert)
```

Comment:
what's the reason for returning true here? I wouldn't expect this to be overridden since the converter sits on the individual properties.

Reply:
@layomia `CanConvert` gets called in `GetConverterFromAttribute` also and returns `typeToConvert == typeof(object)` as default